src/components/markdown-htmlview.component.js

```diff
@@ -102,6 +98,47 @@ const headingRenderer = (node, index, siblings, parent, defaultRenderer) => {
   );
 };
 
+/* eslint-disable no-shadow */
+class CellWithImage extends Cell {
+  render() {
+    const { data, width, height, flex, style } = this.props;
+    let borderWidth;
```

Comment:
Do you think, if you set the variables (`borderWidth`, `borderColor`) to default values, and thus remove the `else` branch?

Reply:
done